dynamic-layers/qt5-layer/recipes-browser/wpewebkit/wpewebkit_%.bbappend

```diff
@@ -0,0 +1,3 @@
+# Enable QT bindings for WPE when meta-qt5 is in layer mix
+#
+PACKAGECONFIG_append = " qtwpe"
```

Comment:
Would it be possible to make this conditional?
Now if a BSP includes meta-qt5, qtwpe is unconditionally built, pulling in qtbase/etc... I don't want this by default.

Reply:
Yeah I think best is to keep it disabled and remove this bbappend